### PR TITLE
[FIX] HTML mode: QUnit.config.noglobals with coverage enabled

### PR DESCRIPTION
**Pull request by matz3**
Code instrumented by istanbul introduces global variables
(cov_* for each file and a global __coverage__ variable).
This causes a failure when the "noglobals" QUnit config is active and
some file is loaded during a test as this pollutes the global namespace.

Hooking into the "after" function allows to add those variables to the
list of known globals so that they won't be detected as new globals.

### CODE REVIEW

File: test/integration/application-proxy/webapp/test/test.qunit.js [hunk 1]
```diff
@@ -5,11 +5,24 @@ QUnit.config.autostart = false;
...
 	"use strict";
 
-	sap.ui.require(["test/app/foo"], function() {
-		QUnit.test("Karma", function(assert) {
-			assert.ok(parent.__karma__.files["/base/webapp/.dotfile"], "Karma files should contain dotfiles");
-		});
-
-		QUnit.start();
+	QUnit.test("Karma", function(assert) {
+		assert.ok(parent.__karma__.files["/base/webapp/.dotfile"], "Karma files should contain dotfiles");
```
Comment (codeworrior) [inline test/integration/application-proxy/webapp/test/test.qunit.js:9]: How is this related to the change?

**Review by codeworrior — APPROVED** [review]
LGTM. It took me a while to understand, how the test works:

- it wasn't obvious to me that coverage is active in the test
- it wasn't obvious to me whether test/app/foo will be instrumented or not
  (because the 'test' folder is excluded from instrumentation - one has to know the difference
   between folder names and module IDs and how both are used in the app)

Well, but the test covers what has been changed, so it's okay.

File: test/integration/application-proxy/webapp/test/test.qunit.js (hunk 1 above)
Comment (matz3) [reply test/integration/application-proxy/webapp/test/test.qunit.js:9]: It's not. The test has been there before.
